sklearn/utils/_testing.py

```diff
@@ -414,7 +414,6 @@ def assert_allclose(
         If None, it is set based on the provided arrays' dtypes.
     atol : float, optional, default=0.
         Absolute tolerance.
-        If None, it is set based on the provided arrays' dtypes.
```

Comment:
Can you open a separate PR for this? This would keep this PR focused on fixing `_check_precomputed_gram_matrix`.

Reply:
Done, see https://github.com/scikit-learn/scikit-learn/pull/23555.